### ISSUE TITLE
Generate base models with items

### ISSUE THREAD

**Issue by charlesfayal** [aws-amplify/amplify-codegen#544]
### Before opening, please confirm:

- [X] I have installed the latest version of the Amplify CLI (see above), and confirmed that the issue still persists.
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-codegen/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-codegen/blob/main/CONTRIBUTING.md#bugs).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### How did you install the Amplify CLI?

npm

### If applicable, what version of Node.js are you using?

18.9.0

### Amplify CLI Version

10.7.3

### What operating system are you using?

Mac

### Amplify Codegen Command

codegen models

### Describe the bug

I'd like to be able to generate typescript models such that they are as they would be when using the API from aws-amplify.
Basically, just creates the base typescript models without lazy models, inits, etc.

Right now it doesn't even create arrays with the items member and nextToken.

### Expected behavior

Create typescript models with the structure that the GraphQL API returns.

### Reproduction steps

amplify codegen models  for any schema

### GraphQL schema(s)

<details>

```graphql
# Put schemas below this line


```

</details>

### Log output

<details>

```
# Put your logs below this line


```

</details>

**Comment by alharris-at** [comment]
Hi @charlesfayal - Thank you for submitting. Marking this as a feature request, and we'll update this ticket as we consider updating the DX for Datastore in the future.
